Makefile.helper.mk

```diff
@@ -15,11 +15,7 @@ KUBECONFIG       ?= ${HOME}/.kube/config
 export PATH := go/bin:$(PATH)
 
 patch:
-	cp .patches/options.patch.go vendor/github.com/google/go-containerregistry/pkg/crane/.
-	cp .patches/getter.patch.go vendor/helm.sh/helm/v3/pkg/getter/.
-	cp .patches/action.patch.go vendor/helm.sh/helm/v3/pkg/action/.
-	cp .patches/install.patch.go vendor/helm.sh/helm/v3/pkg/action/.
-	OUT="$(shell patch -p1 -N -i .patches/helm.patch)" || echo "${OUT}" | grep "Skipping patch" -q || (echo $OUT && false)
+	./scripts/patch.sh
```

Comment:
The Google shell style guide recommends [no extensions for executable scripts](https://google.github.io/styleguide/shellguide.html#file-extensions). Could we follow it here?

Reply:
I don't like this one.

```
Executables should have no extension (strongly preferred) or a .sh extension. Libraries must have a .sh extension and should not be executable.
```

`.sh` seems to be ok.

Also, we already have another script `scripts/rename.sh` and I like the extensions. It also make vim color the files differently according to the file extension.